javalib/src/main/scala/java/util/concurrent/atomic/AtomicLongArray.scala

```diff
@@ -0,0 +1,64 @@
+package io.reactors.common.concurrent
+
+class AtomicLongArray(length: Int) extends Serializable {
+  def this(array: Array[Long]) = {
+    this(array.size)
+    System.arraycopy(array, 0, inner, 0, length)
+  }
+
+  private val inner: Array[Long] = new Array[Long](length)
+
+  final def length(): Int =
+    inner.length
+
+  final def get(i: Int): Long =
+    inner(i)
+
+  final def set(i: Int, newValue: Long): Unit =
+    inner(i) = newValue
+
+  final def lazySet(i: Int, newValue: Long): Unit =
+    set(i, newValue)
+
+  final def getAndSet(i: Int, newValue: Long): Long = {
+    val ret = get(i)
+    set(i, newValue)
+    ret
+  }
+
+  final def compareAndSet(i: Int, expect: Long, update: Long): Boolean = {
+    if (get(i) != expect) false else {
```

Comment:
Our coding style is such that both branches of an if should agree on whether they use braces or not. So since the else branch needs braces, the if branch also takes them:
```scala
if (get(i) != expect) {
  false
} else {
  ...
}
```